### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by matq007**
Fixed minor mistakes and AWS testing!

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/marsseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/marsseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config
Comment (maxulysse) [inline nextflow.config]: 1.0.1

**Review by maxulysse — APPROVED** [review]
Lgtm

File: nextflow.config
Comment (matq007) [reply nextflow.config]: Yes, will run bump-version when I get both approvals :)

**Comment by matq007** [comment]
TODO before merge:

- [x] `bump-version` to `1.0.1`
- [x] update CHANGELOG release date

**Review by JoseEspinosa — APPROVED** [review]
LGTM! :rocket:

File: nextflow.config
Comment (maxulysse) [inline nextflow.config]: more space?

**Review by maxulysse — APPROVED** [review]
:+1:

**Pull request merged**
